package.json

```diff
@@ -99,6 +99,7 @@
     }
   },
   "sideEffects": [
+    "proj.js",
```

Comment:
Does this match a file named `proj.js` anywhere?  Not that this is a problem, just curious.

Reply:
yes: in the npm package, `package.json` is in the same directory as `proj.js`

Reply:
package.json "sideEffects": [
    "proj.js",]
No in branch master !!!!!!

i run 
npm i ol (5.3.0)
And the issue persist then i open the node_modules/ol/package.json and the proj.js is not in sideEffects

Reply:
@leo-117 this change is not included in 5.3.0

Reply:
mmmmm then when ?

Reply:
@leo-117 It will be a while before v6.0. February probably.

Reply:
but this is a fix, not a new functionality

Reply:
in fact, it should be a hot fix

Reply:
In case anyone else needs this: I published a version which contains v5.3.0 + this PR as a hotfix at `@romanzoller/ol`.

Use this version in `package.json`:
```
"@romanzoller/ol": "5.3.0-hotfix-PR-8893"
```

And add an alias for the package, if you are using TypeScript, you can use ["paths"](http://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) inside `"compilerOptions"` in `tsconfig.json`:
```
"baseUrl": ".",
"paths": {
  "ol/*": ["node_modules/@romanzoller/ol/*"]
}
```
(or of course you could just replace all your imports `from 'ol/...'` => `from '@romanzoller/ol/...'`)

Reply:
Thanks for making that fix available @romanzoller.  I just pushed a [v5.3.x branch](https://github.com/openlayers/openlayers/tree/v5.3.x).  If anyone wants to make a pull request with this change applied to that branch, we can cut a patch release.  If you do open a pull request, make sure the [base is set to v5.3.x](https://github.com/openlayers/openlayers/compare/v5.3.x...).

Reply:
Thank you @tschaub, I will make a PR asap.

Reply:
[ol@5.3.1](https://www.npmjs.com/package/ol/v/5.3.1) published.  I haven't updated the website.